utils/jwtUtil.js

```diff
@@ -90,12 +90,12 @@ function getUsernameFromToken(req) {
     if (req && req.headers && req.headers.authorization) {
       jwt.verify(req.headers.authorization, env.tokenSecret, {},
       (err, decodedData) => {
-        if (err !== null) {
-          reject(err);
+        if (err !== null || !decodedData) {
+          return reject(err);
```

Comment:
@shriramshankar Do we need to return reject or resolve? According to MDN, the reject/resolve returns the corresponding promise.